### ISSUE TITLE
Create a new release

### ISSUE THREAD

**Issue by SunburtReynolds** [corymsmith/react-native-fabric#187]
Hey @corymsmith! Any chance you could create a new release? There have been a few critical updates merged into master since last August that my team really need. Specifically, we need to run the most recent version of React Native (0.56) which requires Android SDK 26. A commit to support this version of RN was merged here: https://github.com/corymsmith/react-native-fabric/commit/2df03b6b8b1e65b639b050241b89bb97f3c050be. We just need your help to create a release :)

Thanks so much.

**Comment by gabceb** [comment]
For anyone waiting on this issue, friendly reminder you can add a dependency directly from Github while @corymsmith cooks a release for us.

using yarn `yarn add https://github.com/corymsmith/react-native-fabric#49958e731e978eb09ca29e84388743b8c2440ce8`

Also friendly reminder that GH reactions do no send notifications so they will mostly be ignored by the maintainers unless they come back to the issue. Not saying you should spam but something to consider if you want to have their attention 😏

**Comment by skb1129** [comment]
A new release is required!!!!!
